app/Providers/AuthServiceProvider.php

```diff
@@ -181,7 +181,8 @@ public function boot()
                 || $user->can('checkin', Asset::class)
                 || $user->can('audit', Asset::class)       
                 || $user->can('update', License::class)   
-                || $user->can('create', License::class)   
+                || $user->can('create', License::class)
+                || $user->can('delete', License::class)
```

Comment:
Checking in a license seat isn't actually the same as deleting the license itself, so I think this logic might not be necessary.